stackslib/src/net/download/nakamoto/tenure_downloader.rs

```diff
@@ -558,10 +581,11 @@ impl NakamotoTenureDownloader {
         response: StacksHttpResponse,
     ) -> Result<Option<Vec<NakamotoBlock>>, NetError> {
         let handle_result = match self.state {
-            NakamotoTenureDownloadState::GetTenureStartBlock(_block_id) => {
+            NakamotoTenureDownloadState::GetTenureStartBlock(_block_id, _start_request_time) => {
```

Comment:
Nit here (and a few other similar lines) - don't prefix with `_` if it gets used